### ISSUE TITLE
The wrong Taxonomy compilation command

### ISSUE THREAD

**Issue by roytman** [fybrik/airbyte-module#87]
The file [https://github.com/fybrik/airbyte-module/tests/test_fybrik.sh](https://github.com/fybrik/airbyte-module/tree/main/tests/test_fybrik.sh) uses the old fybrik taxonomy commands:
```
go run main.go taxonomy compile --out custom-taxonomy.json --base charts/fybrik/files/taxonomy/taxonomy.json$AIRBYTE_FYBRIK_TEST/fybrik-taxonomy-customize.yaml
```

As expected, it fails, 
```
Error: unknown flag: --out
Usage:

Flags:
      --config string   config file (default is $HOME/.fybrik.yaml)
  -h, --help            help for fybrik

Error: unknown flag: --out
```

**Comment by revit13** [comment]
@cdoron can `tests/test_fybrik.sh` file be removed? We already have a test under `hack/test_module.sh` that is used for testing which seems to be identical. Thanks

**Comment by cdoron** [comment]
The `hack/test_module.sh` test writes and reads from a MySQL database.
The `tests/test_fybrik.sh` test reads from a parquet file it reads from `https://github.com/Teradata/kylo/raw/master/samples/sample-data/parquet/userdata2.parquet`.
I think that there is a place for both tests.
Fixed by #88
